### PR TITLE
Contact form

### PR DESCRIPTION
**Pull request by houdamzari**
- Added contact form 
- Added validation rules to the form
- Setup Formspree

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -248,8 +249,20 @@ <h1>About <br />Myself</h1>
...
+        <form action="https://formspree.io/f/mjvzpbgj" method="post">
+            <h1 class="form_header">Contact me</h1>
+            <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
+                
+            <div class="form_inputs">
+                <input type="text" name="name" placeholder="Name" class="form_input" maxlength="30" required class="form_input">
+                <input type="email" name="email" placeholder="Email" class="form_input" required class="form_input">
+                <textarea name="message" id="" cols="30" rows="10" maxlength="500" placeholder="Write your message here" class="form_input" required></textarea>
+            </div>
+          <button type="submit" class="form_button">Get in Touch</button>
```
Comment (Olamarx) [inline index.html:253]: - [ ] Kindly add the button interaction in terms of hover and others just as on the Figma template
image

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @houdamzari && @Lucash2022, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✔️ 
- Form works as required ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Olamarx — CHANGES_REQUESTED** [review]
Hi @houdamzari && @Lucash2022, 

Good job so far, indeed you have spent time on this project!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights
- All linters passed ✔️ 
- Form works as required ✅ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: index.html [hunk 2]
```diff
@@ -252,17 +252,17 @@ <h1>About <br />Myself</h1>
       </ul>
     </section>
     <footer class="footer">
-      <form action="" method="post">
-          <h1 class="form_header">Contact me</h1>
-          <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
-              
-          <div class="form_inputs">
-              <input type="text" placeholder="Name" class="form_input" maxlength="30" required class="form_input">
```
Comment (Talha-Hanif5666) [inline index.html]: Nice work though, but i think it is better not to leave any blank lines in your code where unnecessary.  Doing this your code looks your code unclear.
Kindly please remove that blank line. Thank you

**Comment by leehaney254** [comment]
Your commits are professional and descriptive. Nice work :smile:

**Review by Talha-Hanif5666 — COMMENTED** [review]
a

File: index.html [hunk 3]
```diff
@@ -248,8 +239,19 @@ <h1>About <br />Myself</h1>
         </ul>
         <div class="dropdown"><li class="section_header">Frameworks</li><span>^</span></div>
         <div class="dropdown"> <li class="section_header">Skills</li><span>^</span>
-       
       </ul>
     </section>
+    <footer class="footer">
+        <form action="https://formspree.io/f/mjvzpbgj" method="post">
+            <h1 class="form_header">Contact me</h1>
+            <p class="contact_paragraph">If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I’d love to help with it </p>
```
Comment (Bondok6) [inline index.html:247]: - [OPTIONAL]  I suggest removing the `<h1>` tag from here and adding another hx (`<h2>`, `<h3>`, ....) 👌 . to Follow best practices for HTML **Only Use H1 Elements Once Per Page for optimal SEO**. 
for more info, you can check this [Link](https://techstacker.com/seo-only-use-h1-elements-once-per-page/). 💡

**Review by Bondok6 — APPROVED** [review]
# STATUS: APPROVED :green_circle: :clap: 

## Hi @houdamzari  and @Lucash2022 🙋‍♂️, 
Your project is complete!  There is nothing else to say other than... Great job 👏 
Congratulations! 🎉 

TO HIGHLIGHT:

- Great job Implementing the changes provided by the previous reviewers ✅
- Your project is professional ✅
- Your repo is professional ✅
- Well structured files ✅
- No Linter errors ✅


Now you can merge this branch and move on :1st_place_medal: 
HAPPY CODING AND GOOD LUCK WITH THE COMMING ONES :crossed_fingers: :four_leaf_clover: 




## Optional suggestions

- [OPTIONAL] I suggest adding the `MIT.md` file to your project 👌, Here is the [Micorverse Template](https://github.com/microverseinc/readme-template/blob/master/MIT.md), and remember to customize it 😉

> The MIT license gives users express permission to reuse code for any purpose, sometimes even if the code is part of proprietary software. As long as users include the original copy of the MIT license in their distribution, they can make any changes or modifications to the code to suit their own needs. 💯

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**